src/main/resources/metadata/tezos.mainnet.conf

```diff
@@ -338,9 +338,94 @@
         }
       }
     }
+    big_maps {
+      display-name-plural: "Big Maps"
+      display-name: "Big Map"
+      visible: true
+      attributes {
+        big_map_id {
+          display-priority: 0
+          display-order: 0
+          display-name: "Id"
+          data-type: "int"
+          visible: true
+        }
+        key_type {
+          display-priority: 0
+          display-order: 1
+          visible: true
+        }
+        value_type {
+          display-priority: 0
+          display-order: 2
+          visible: true
+        }
+      }
+    }
+    big_map_contents {
+      display-name-plural: "Big Map Contents"
+      display-name: "Big Map Content"
+      visible: true
+      attributes {
+        big_map_id {
+          display-priority: 0
+          display-order: 3
+          display-name: "Map Id"
+          data-type: "int"
+          visible: true
+          reference: {
+            entity: "big_maps"
+            key: "big_map_id"
+          }
+        }
+        key {
+          display-priority: 0
+          display-order: 0
+          visible: true
+        }
+        key_hash {
+          display-priority: 0
+          display-order: 1
+          visible: true
+        }
+        value {
+          display-priority: 0
+          display-order: 2
+          visible: true
+        }
+      }
+    }
+    originated_account_maps {
+      display-name-plural: "Contract Big Maps"
+      display-name: "Contract Big Maps"
+      visible: true
+      attributes {
+        big_map_id {
+          display-priority: 0
+          display-order: 0
+          display-name: "Map id"
+          data-type: "int"
+          visible: true
+          reference: {
+            entity: "big_maps"
+            key: "big_map_id"
+          }
+        }
+        account_id {
+          display-priority: 0
+          display-order: 1
+          data-type: "accountaddress"
+          visible: true
+          reference: {
+            entity: "accounts"
+            key: "account_id"
+          }
+        }
+      }
+    }
     blocks {
-      display-name-plural: "Blocks"
-      display-name: "Block"
+      display-name-plural: "blocks"
```

Comment:
why did you change it to lowercase only here?

Reply:
I guess it must've been an unwilling change that slipped by... thanks

Reply:
I think it might've been some semi-automated formatting that I didn't even expect to be there